db/migrate/20170711063338_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[5.1]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :email
+      t.string :password
```

Comment:
so u are storing passwords as plain text and every admin can read them in console?